docs/INSTALL.TXT

```diff
@@ -5,19 +5,20 @@ Welcome to Zen Cart(R) E-Commerce
 THE FOLLOWING IS A VERY SIMPLIFIED INSTALLATION DOCUMENT.
 You can find online documentation at: https://docs.zen-cart.com
 
-WE SUGGEST YOU USE THE PDF "Implementation Guide" PDF FILE IN THE /docs FOLDER FOR FULLY DETAILED INSTRUCTIONS.
+WE SUGGEST YOU USE ONLINE DOCS AND THE PDF "Implementation Guide" 
+
+https://docs.zen-cart.com/release/implementation-guide-v157.pdf
```

Comment:
Should this be implementation-guide-v158.pdf.
I know it does not exist but may be confusing if implementing 1.5.8 so would need creating and minor mods

Reply:
Thanks. I think we'll just leave off the version number moving forward.

Reply:
Thats fine. The PDF mentions 1.5.7 so will also need changing if you are going to leave it off.